src/current/_includes/metrics-usage/tenant.consumption.external_io_egress_bytes.md

```diff
@@ -2,4 +2,4 @@ The number of RUs consumed by byte traffic for cluster bulk I/O operations (e.g.
 <br>
 <b>1 RU = 1 KiB Network egress.</b>
 <br>
-Correlate this metric with <a href="#tenant.consumption.request_units">Request Units (RUs)</a>. To learn more about how RUs are calculated, refer to <a href="https://www.cockroachlabs.com/docs/cockroachcloud/serverless-resource-usage">Resource Usage</a>.
\ No newline at end of file
+Correlate this metric with <a href="#tenant.consumption.request_units">Request Units (RUs)</a>. To learn more about how RUs are calculated, refer to [Resource Usage]({% link cockroachcloud/resource-usage-basic.md %}).
```

Comment:
According to the [Cloud 2.0 docs plan](https://docs.google.com/document/d/1__C54ZiDlnaDYddnOjDEemadNIOGaJODw_RHMxNaytc/edit), we should be linking to Standard by default, rather than Basic. The Request Units page/metrics also seem to apply specifically to Standard. So should these actually link to `resource-usage.md`? Sorry if I missed something obvious.

Reply:
see [comment](https://github.com/cockroachdb/docs/pull/18877#issuecomment-2327122179)